avocado/core/task/statemachine.py

```diff
@@ -253,7 +251,7 @@ async def start(self):
         if start_ok:
             LOG.debug('Task "%s": spawned successfully',
                       runtime_task.task.identifier)
-            runtime_task.status = None
+            runtime_task.status = "RUNNING"
```

Comment:
This change sounds odd. Could you please elaborate why are you doing this in the commit message?